src/components/augmint-ui/container/styles.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+import { media } from "styles/media";
+
+const BaseDiv = `
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 1127px;
```

Comment:
these `max-width`s seem like magic numbers... did you consider moving them to [styles](https://github.com/Augmint/augmint-web/tree/staging/src/styles)?